rllib/algorithms/cql/torch/cql_torch_learner.py

```diff
@@ -144,15 +150,24 @@ def compute_loss_for_module(
         # Calculate the TD error.
         td_error = torch.abs(q_selected - q_selected_target)
         # TODO (simon): Add the Twin TD error
+        if config.twin_q:
+            td_error += torch.abs(q_twin_selected, q_selected_target)
+            # Rescale the TD error
+            td_error += 0.5
```

Comment:
Should this be `* 0.5`?

Reply:
Great catch :D This should be multiplied by 0.5